dockers/broad/dragmap/docker_build.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Update version when changes to Dockerfile are made
-DOCKER_IMAGE_VERSION=1.0.0
+DOCKER_IMAGE_VERSION=1.0.1
```

Comment:
Maybe this should be more like a '1.1.0' version change?

Reply:
im cool with whatever, just figured since id go with that since it was a security patch